src/index.html

```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html>
     <body>
+       
         <p>Hello World!</p>
+        <div>
+            <button onclick="getTimeline()">Get Timeline</button>
+            <p id="timeline-para"></p>
+        </div>
+
+        <script type="text/javascript" src="src/js/basicRequests.js"></script>
```

Comment:
put this in the <head> tag

Reply:
I'm sorry, I'm not sure I understand which tag you are referring to, or what should be in it. Is the script supposed to be elsewhere?

Reply:
<head>

Reply:
head tag

Reply:
read up about what goes in the head tag and please adjust as necessary